WordPress/openverse
# Add source, creator and tag links to the single result page media info

## Fixes



Fixes #2730 by @obulat

## Description


This PR updates the single result page's creator, source and tag links to the new designs, linking them to the collection pages. 

### Audio
<img width="840" alt="Screenshot 2023-11-13 at 12 33 00 PM" src="https://github.com/WordPress/openverse/assets/15233243/dc96ae3f-6d04-4a3e-ba3a-052db6e09407">

### Image 
<img width="842" alt="Screenshot 2023-11-13 at 12 34 38 PM" src="https://github.com/WordPress/openverse/assets/15233243/f9bd70cf-0848-4ed3-8d79-8003cd8da3ce">

If the creator and source names are longer than the page width, they become scrollable. Setting up the scrolling to work in both LTR and RTL was a bit complex.

Note that the link URLs will be updated after #3140 is merged because it's impossible to add a link to a localized page that does not exist (and we don't currently have pages for `source`/`creator`/`tag` collections).

## Testing Instructions

Run the app
Go to localhost:8443/preferences and switch the `additional_search_views` flag `on`
Go to single result pages for audio and image. 
The source, creator and tag should have an updated style with correct links (`/image/tag/cat`, `/image/source/flickr`, `/image/source/flickr/creator/name`). Note that i18n (linking to the correct locale if you are not on the English one) does not work yet because the pages for tag/source/creator do not exist yet. The links also do not open the pages because the pages are created in #3140 
When the creator and the source name are longer than the available width, they become scrollable. To check, see http://localhost:8443/image/8c8f2318-acc8-477c-a059-34f44901933e?q=cat

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
